blogposts/2021/dev-tool-time-roger-peppe.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Simplicity is worth paying for: Dev Tool Time with Roger Peppé'
+description: 'Roger Peppé, Software Engineer at InfluxData, shares his standing desk setup, and how the Acme text editor and mouse chording drive his coding workflow.'
+author: Scott Bailey
+authorUrl: https://about.sourcegraph.com/handbook/company/team#scott-bailey-he-him
+publishDate: 2021-11-17T10:00-07:00
+tags: [blog]
+slug: dev-tool-time-roger-peppe
+heroImage: https://sourcegraphstatic.com/blog/dev-tool-time/roger-peppe/linkedin.png
+socialImage: https://sourcegraphstatic.com/blog/dev-tool-time/roger-peppe/linkedin.png
+published: true
+---
+
+Many devs, especially those who love Vim and Emacs, carefully fine-tune their editor, terminal, and even OS configurations so that their hands rarely leave the keyboard. Roger Peppé, Software Engineer at InfluxData, took a different path, driven by the simplicity and power of the [Acme text editor](http://acme.cat-v.org/) and mouse chording. When Rog joined us for [Dev Tool Time](https://info.sourcegraph.com/dev-tool-time) on November 10, 2021, he powerfully demonstrated a different model of text editing, which will seem at once familiar yet strange.
```

Comment:
```suggestion
Many devs, especially those who love Vim and Emacs, carefully fine-tune their editor, terminal, and even OS configurations so that their hands rarely leave the keyboard. Roger Peppé, Software Engineer at InfluxData, took a different path, driven by the simplicity and power of the [Acme text editor](http://acme.cat-v.org/) and mouse chording. When Rog joined us for [Dev Tool Time](https://info.sourcegraph.com/dev-tool-time) on November 10, 2021, he demonstrated a different model of text editing, which will seem at once familiar and strange.
```